apps/indexer/src/management/migrations.ts

```diff
@@ -177,6 +178,210 @@ class StaticMigrationsProvider implements MigrationProvider {
           await db.schema.dropTable("comment_reports").execute();
         },
       },
+      "2025_07_24_09_18_00_add_maintenance_functions": {
+        up: async (db: Kysely<IndexerSchemaDB>) => {
+          await db.executeQuery(
```

Comment:
this code needs comments - what's it doing and why

Reply:
added comments.